spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -498,6 +499,14 @@ def and_the_step_by_step_is_not_editable
     there_should_be_no_unpublish_button
   end
 
+  def but_the_user_can_still_add_change_notes
```

Comment:
This is an entire "add a change note" test, not a step. The change note test is in `managing_steps_spec` which is probably not the best place for it, but it probably doesn't need to be tested again here. 

I think this step should be split into two:

`when_I_visit_the_change_notes_page`
`then_I_can_add_a_change_note` - This really only needs to check that the add change note buttons are visible.